addons/html_editor/static/src/main/toolbar/toolbar_plugin.js

```diff
@@ -348,20 +381,18 @@ export class ToolbarPlugin extends Plugin {
         }
         const nodes = this.getFilterTraverseNodes();
         for (const buttonGroup of this.buttonGroups) {
-            if (buttonGroup.namespace === this.state.namespace) {
-                for (const button of buttonGroup.buttons) {
-                    this.state.buttonsActiveState[button.id] = button.isActive?.(selection, nodes);
-                    this.state.buttonsDisabledState[button.id] = button.isDisabled?.(
-                        selection,
-                        nodes
-                    );
-                    this.state.buttonsAvailableState[button.id] =
-                        button.isAvailable === undefined || button.isAvailable(selection);
-                    this.state.buttonsTitleState[button.id] =
-                        button.title instanceof Function
-                            ? button.title(selection, nodes)
-                            : button.title;
+            for (const button of buttonGroup.buttons) {
+                if (!button.namespaces.includes(this.state.namespace)) {
+                    continue;
```

Comment:
I hate `continue` but it's entirely subjective. I would like it if it was called `skip`, but `continue` just seems like "ok, then I'll continue executing further", which is exactly the opposite of what it does. I get that it "continues" executing the `for` loop itself and not the block inside of it, but it's just very confusing and I have to wrap my head around that particular instruction every single time I see it. I blame whoever decided to name this thing `continue`. Just wanted to rant about it, I understand that it lets you gain one indentation level.

Reply:
`continue` keyword comes from C, which is a WONDERFUL language. :smile: (no trolling!)
 Would you rather me move everything into a `if` statement? I would be ok with that. But please note that there are 39 other occurrences of `continue;` in html_editor...